airflow/cli/commands/dag_command.py

```diff
@@ -378,6 +379,47 @@ def dag_list_dag_runs(args, dag=None):
     print(table)
 
 
+@cli_utils.action_logging
+def generate_pod_yaml(args):
+    """Generates yaml files for each task in the DAG. Used for testing output of KubernetesExecutor"""
+
+    from airflow.executors.kubernetes_executor import AirflowKubernetesScheduler, KubeConfig
+    from airflow.kubernetes import pod_generator
+    from airflow.kubernetes.pod_generator import PodGenerator
+    from airflow.kubernetes.worker_configuration import WorkerConfiguration
+    execution_date = datetime.datetime(2020, 11, 3)
```

Comment:
Why is the execution_date hardcoded ?

Reply:
@kaxil since this endpoint is for testing purposes, it didn't make sense to force users to put in an execution date, and it seemed to make sense to hard code it so there would be consistency. Do you feel differently?

Reply:
Elsewhere execution_date is required. Should we unify it?
```
usage: airflow tasks test [-h] [-n] [--env-vars ENV_VARS] [-m] [-S SUBDIR]
                          [-t TASK_PARAMS]
                          dag_id task_id execution_date
```
```
airflow dags test [-h] [--imgcat-dagrun] [--save-dagrun SAVE_DAGRUN]
                         [--show-dagrun] [-S SUBDIR]
                         dag_id execution_date
```
```
usage: airflow tasks run [-h] [--cfg-path CFG_PATH] [-f] [-A] [-i] [-I] [-N]
                         [-l] [-m] [-p PICKLE] [--pool POOL] [--ship-dag]
                         [-S SUBDIR]
                         dag_id task_id execution_date
```